spec/lib/http/uri_spec.rb

```diff
@@ -11,6 +11,92 @@
   subject(:https_uri) { described_class.parse(example_https_uri_string) }
   subject(:ipv6_uri) { described_class.parse(example_ipv6_uri_string) }
 
+  describe "NORMALIZER" do
```

Comment:
Can you please move it to a separate spec file `spec/lib/http/uri/normalizer_spec.rb` instead? So it will look like:

``` ruby
RSpec.describe HTTP::URI::NORMALIZER do
  ...
end
```